airflow/providers/databricks/hooks/databricks.py

```diff
@@ -656,14 +655,20 @@ def get_repo_by_path(self, path: str) -> str | None:
 
         return None
 
-    def update_job_permission(self, json: dict[str, Any]) -> dict:
+    def update_job_permission(self, job_id: int, json: dict[str, Any]) -> dict:
         """
         Update databricks job permission.
 
+        :param job_id: job id
         :param json: payload
         :return: json containing permission specification
         """
-        return self._do_api_call(UPDATE_PERMISSION_ENDPOINT, json)
+        response = {}
+        try:
+            response = self._do_api_call(("PATCH", f"api/2.0/permissions/jobs/{job_id}"), json)
+        except Exception as e:
+            print("Could not update permission ", e)
```

Comment:
I think we probably should use logging instead of print or reraise?

Reply:
I guess reraise makes more sense in this case

Reply:
I have added logs instead of rerasing any exception, so that it does not disrupt original flow.